ru/turing/courses/lesson3/Main.java

```diff
@@ -0,0 +1,25 @@
+package ru.turing.courses.lesson3;
+
+import ru.turing.courses.lesson3.objects.Developer;
+import ru.turing.courses.lesson3.objects.PremiumPlayer;
+import ru.turing.courses.lesson3.objects.StandartPlayer;
+
+public class Main {
+    public static void main(String[] args) {
+        Registry<StandartPlayer> standartPlayerRegistry = new Registry<StandartPlayer>();
+        Registry<PremiumPlayer> premiumPlayerRegistry = new Registry<PremiumPlayer>();
+        Registry<Developer> developerRegistry = new Registry<Developer>();
+
+
+        StandartPlayer player_1 = new StandartPlayer("p_001", "Nagibator2010");
```

Comment:
```suggestion
        StandartPlayer playerOne = new StandartPlayer("p_001", "Nagibator2010");
```
CamelCase! И в остальных местах тоже исправь